src/Product/SearchProvider.php

```diff
@@ -202,7 +202,7 @@ public function renderFacets(ProductSearchContext $context, ProductSearchResult
             return '';
         }
 
-        return $this->module->render(
+        return $this->module->fetch(
             'views/templates/front/catalog/facets.tpl',
```

Comment:
Why not `module:ps_facetedsearch/` here?

Reply:
Hum this IS strange. I've replaced it. It need on Évry fetch